framework/src/modules/chain/blocks/block.js

```diff
@@ -23,7 +23,6 @@ const {
 	verifyData,
 } = require('@liskhq/lisk-cryptography');
 const _ = require('lodash');
-const crypto = require('crypto');
```

Comment:
If we're not using anymore this library we should remove it from the tests as well and ultimately from the `package.json`. Is that feasible @lsilvs ?